docs/2.10.0/docs/canton/usermanual/upgrading.rst

```diff
@@ -223,6 +251,14 @@ command.
 Note that the ``migrate_domain`` command in 2.9 now automatically removes such data,
 but only for the sync domain on which it has been invoked.
 
+Paging in party Management
+""""""""""""""""""""""""""
+By default, the `ListKnownParties` method on the `PartyManagementService` will now only return at most 10'000
+to avoid memory issues in participants that knows more than 10'000 parties.
```

Comment:
```suggestion
The `ListKnownParties` method on the `PartyManagementService` now returns at most 10,000 results by default
to avoid memory issues in participants that know more than 10,000 parties.
```